### PR TITLE
feat: import account from private key

### LINKED ISSUE #591: feature request: import private key
### Motivation

It would be nice to be able to import a single account using a private key.

### Usage example

This is especially helpful for testing with a local instance of Fuel Node running.

### Possible implementations

Similar to how metamask allows the same functionality.

### PR DESCRIPTION
**Pull request by LuizAsFight**
Closes #591

### CODE REVIEW

File: packages/app/src/systems/Account/hooks/useAccountFormNameObj.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import type { Account } from '@fuel-wallet/types';
+import { useMemo } from 'react';
+import * as yup from 'yup';
+
+export function useAccountFormNameObj(accounts: Account[]) {
```
Comment (luizstacio) [inline packages/app/src/systems/Account/hooks/useAccountFormNameObj.tsx:5]: Why NameObj on the end?

File: packages/app/src/systems/Account/machines/accountsMachine.tsx [hunk 2]
```diff
@@ -333,14 +363,53 @@ export const accountsMachine = createMachine(
...
+          }
+          if (!input?.name.trim()) {
+            throw new Error('Name cannot be empty');
+          }
+
+          // Add account to vault
+          const accountVault = await VaultService.createVault({
+            type: 'privateKey',
+            secret: input.privateKey,
+          });
```
Comment (luizstacio) [inline packages/app/src/systems/Account/machines/accountsMachine.tsx:392]: You don't need to create a new vault for each privateKey a vault of type privateKey can handle multiple different privateKeys.

File: packages/app/src/systems/Core/utils/string.tsx [hunk 3]
```diff
@@ -21,3 +21,41 @@ export const truncate = (str: string, length: number = 30) => {
...
+    const repeatedAssets = allValues.filter(
+      (a) => a && a.trim() === value.trim()
+    );
+
+    let notRepeatedField: string;
+    if (repeatedAssets[0]) {
+      const nextToTry =
+        tries === 1
+          ? `${value} (${tries})`
+          : `${value?.slice(0, -4)} (${tries})`;
```
Comment (luizstacio) [inline packages/app/src/systems/Core/utils/string.tsx:49]: Why we need this slice here?

**Review by luizstacio — CHANGES_REQUESTED** [review]

File: packages/app/src/systems/Core/utils/string.tsx (hunk 3 above)
Comment (LuizAsFight) [reply packages/app/src/systems/Core/utils/string.tsx:49]: remove last (1) and inform new (2) for example.

Comment (luizstacio) [reply packages/app/src/systems/Core/utils/string.tsx:49]: And if the number is `Text (10)` What happens?

Comment (LuizAsFight) [reply packages/app/src/systems/Core/utils/string.tsx]: good catch my friend

File: packages/app/src/systems/Account/machines/accountsMachine.tsx (hunk 2 above)
Comment (luizstacio) [reply packages/app/src/systems/Account/machines/accountsMachine.tsx:392]: https://github.com/FuelLabs/fuels-wallet/issues/617

**Review by luizstacio — DISMISSED** [review]

**Review by luizstacio — APPROVED** [review]

**Pull request merged**